cmake/modules/thirdparty/FindDotnet.cmake

```diff
@@ -137,6 +138,16 @@ EXECUTE_PROCESS(
     OUTPUT_STRIP_TRAILING_WHITESPACE
 )
 
+EXECUTE_PROCESS(
+    COMMAND ${DOTNET_EXE} --list-sdks
```

Comment:
`${DOTNET_EXE}` should be in double quotes, to protect embedded space. You know `C:\Program Files...`.

This is something to keep in mind more generally for single values in CMake (but not always for lists of values). e.g. `DOTNET_SDKS` below. I'm aware that this insufficiently implemented in this file and in many other cmake files here.

Reply:
Good catch. Fixed in the latest update.

Interestingly, the original code that was written by Microsoft had the same bug ... :)